.deploy/deploy.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+function log_error_and_exit () {
+    local LAST_COMMAND=$1
+    local LAST_STATUS=$2
+    logger "Error running $0; received ${LAST_STATUS} from ${LAST_COMMAND}"
+    exit ${LAST_STATUS}
+}
+
+echo "[BEGIN] deploy apigility.org"
```

Comment:
I believe we want to update here - no more apigility.org

Reply:
This entire directory is obsolete with the move to platform.sh. I'll remove it in a subsequent patch.